wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Commands.java

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.wpilibj2.command;
+
+import static edu.wpi.first.wpilibj.util.ErrorMessages.requireNonNullParam;
+
+import java.util.Map;
+import java.util.function.BooleanSupplier;
+import java.util.function.Supplier;
+
+/**
+ * Namespace for command factory methods.
+ *
+ * <p>For convenience, you might want to static import the members of this class.
+ */
+public enum Commands {
+// no enum entries - utility class
```

Comment:
I don't think we do this anywhere else.  Generally our utility classes are "normal" classes with a private constructor that throws.